test_cases/README.md

```diff
@@ -52,6 +52,15 @@ annotations,
 [this page](https://typing.readthedocs.io/en/latest/source/quality.html#testing-using-assert-type-and-warn-unused-ignores)
 provides a useful guide.
 
+### Naming convention
+
+Use the same top-level name for the module / package you would like to test.
+Use `test_${thing}.py` naming pattern for individual test files.
+For example:
+- Tests for `gather` defined in `stdlib/asyncio/tasks.pyi` will go into `stdlib/asyncio/test_gather.py`
```

Comment:
```suggestion
- Tests for `gather` defined in `stdlib/asyncio/tasks.pyi` will go into `stdlib/asyncio/tasks/test_gather.py`
```

Reply:
Should we move it then? https://github.com/python/typeshed/blob/master/test_cases/stdlib/asyncio/test_gather.py